pykafka/connection.py

```diff
@@ -18,7 +18,9 @@
 limitations under the License.
 """
 __all__ = ["BrokerConnection"]
+from functools import partial
 import logging
+import ssl
```

Comment:
Hahaha, uhm, I suppose if I were to confess that I thought I'd done that, that'd be kind of embarassing, right? So, no confessions :)